Sources/Phakchi/Encodable.swift

```diff
@@ -1,52 +1,63 @@
 import Foundation
 
-typealias JSONObject = [String: AnyObject]
+typealias JSONObject = [String: PactEncodable]
 
 public protocol PactEncodable {
-    var pactJSON: AnyObject { get }
+    var pactJSON: PactEncodable { get }
 }
 
 extension PactEncodable {
-    var JSONData: NSData {
-        if let data = try? NSJSONSerialization.dataWithJSONObject(pactJSON, options: []) {
+    var JSONData: Data {
+        if let data = try? JSONSerialization.data(withJSONObject: pactJSON, options: []) {
             return data
         }
         fatalError("Could not deserialize JSON")
     }
 }
 
 extension Int: PactEncodable {
-    public var pactJSON: AnyObject {
+    public var pactJSON: PactEncodable {
         return self
     }
 }
 
 extension Double: PactEncodable {
-    public var pactJSON: AnyObject {
+    public var pactJSON: PactEncodable {
         return self
     }
 }
 
 extension Bool: PactEncodable {
-    public var pactJSON: AnyObject {
+    public var pactJSON: PactEncodable {
         return self
     }
 }
 
 extension String: PactEncodable {
-    public var pactJSON: AnyObject {
-        return self
+    public var pactJSON: PactEncodable {
+        return self as PactEncodable
```

Comment:
I'm surprised you would need `as PactEncodable` even though you declared it implementing `PactEncodable` just above.